artview/components/select_region.py

```diff
@@ -13,7 +14,7 @@
 
 from ..core import (Variable, Component, common, VariableChoose,
                     componentsList, QtGui, QtCore)
-from ..core.points import Points, write_points_csv, read_points_csv
+from ..core.points import Points
 
```

Comment:
If you want a fast merge we can do it.

I will just take some time till I get back to the code for correcting some of the points here. I will open a PR when it is ready.